optitrack_listener:
  ros__parameters:
    topic_name: "/natnet_ros/Robot_1/pose"